app/views/documents/fields/_govspeak_input.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds" id="<%= "#{field.id}" %>">
```

Comment:
Oh shoot I told you this totally wrong earlier didn't I? Anyway very minor but this doesn't need interpolation It can be: `<%= field.id %>`

Reply:
Done